eng/build/Release.targets

```diff
@@ -0,0 +1,33 @@
+<Project>
+
+  <!--
+    Release.targets: targets relating to the release integrity of our product.
+    - Nuget packaging targets
+    - Versioning targets
+  -->
+
+  <PropertyGroup>
+    <BeforePack>$(BeforePack);GetReleaseNotes;AddTagToReleaseNotes</BeforePack>
+  </PropertyGroup>
+
+  <!-- This target lets us define a release notes file per package. -->
```

Comment:
Do we want to consider a similar approach to the Azure SDK repo with one long running change log file?

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/CHANGELOG.md

Reply:
I view `CHANGELOG.md` as a parallel solution to `RELEASE_NOTES.md`.

Changelog is long running and release_notes is for an individual release. It may be valuable for us to include both.